components/blocks/nav/SiteNavbar/SiteNavbar.vue

```diff
@@ -106,14 +93,14 @@ function logout() {
           class="shadow-xl"
           :modal="true"
           @close="state.showModal = false"
-          @signUpClick="state.showLogin = false"
+          @sign-up-click="state.showLogin = false"
```

Comment:
💭 I assume the lowercase dash syntax is the preferred syntax here, over what I had previously?

Reply:
This must have been auto done, but ya `this-type-of-syntax` is standard for this